Xamarin.Forms.Platform.Android/Renderers/ButtonRenderer.cs

```diff
@@ -260,6 +263,24 @@ void UpdateTextColor()
 			_textColorSwitcher?.UpdateTextColor(Control, Element.TextColor);
 		}
 
+		Thickness paddingDelta = new Thickness();
+
+		void UpdatePadding()
```

Comment:
These changes need to be applied to the AppCompat Renderer and the Fast Renderer 

https://github.com/xamarin/Xamarin.Forms/blob/b96f65bdd9e7e931971338b122a8940e2a35ccc7/Xamarin.Forms.Platform.Android/AppCompat/ButtonRenderer.cs
https://github.com/xamarin/Xamarin.Forms/blob/b96f65bdd9e7e931971338b122a8940e2a35ccc7/Xamarin.Forms.Platform.Android/FastRenderers/ButtonRenderer.cs